railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_5_1.rb.tt

```diff
@@ -5,6 +5,9 @@
 # Once upgraded flip defaults one by one to migrate to the new default.
 #
 # Read the Guide for Upgrading Ruby on Rails for more info on each option.
+
+# Make `form_with` generate non-remote forms.
+Rails.application.config.action_view.keep_form_with_remote = false
```

Comment:
What keep form with remote means?

That all forms generated with form with is generated with remote true?